backend/src/main/resources/application.properties

```diff
@@ -16,5 +16,7 @@ server.port=8010
 
 #Needed due to empty return msg from json
 server.error.include-message=always
+
 #JWT
 jwt.jwtExpirationMs=3600000
+jwt.jwtSecret=9z$C&F)J@McQfTjWnZr4u7x!A%D*G-Ka
```

Comment:
Shouldn't this be detoriated from an .env file or smth?
It looks... wrong as an inline chage.

Reply:
what do you mean? key secret should be in properties

Reply:
Will be changed with https://github.com/Inzynierowie/Inzynierka/issues/52